### PR TITLE
New structure

### PR DESCRIPTION
**Pull request by gooklani**
This PR introduces a new structure for the HTML boilerplate template.

### CODE REVIEW

File: src/pages/partials/_footer.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<footer class="footer content-wrapper padding-inline-normal padding-top-60 tablet-text-center">
+   <hr/>
+   <div class="footer__content color-light padding-block-normal">
+      <div>
+         <p>
+            © 2015 - 
+            <span id=&#8221;current-year&#8221;>
```
Comment (mahdiyazdani) [inline src/pages/partials/_footer.html:7]: Date could be retrieved dynamically from the `scripts.js` file.

File: src/pages/partials/_header.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<header class="header background-dark color-tile has-color">
+   <div class="header__content content-wrapper padding-inline-normal padding-block-small">
+      <a class="header__brand" href="../">
```
Comment (mahdiyazdani) [inline src/pages/partials/_header.html:3]: Href value might require double checking as Github handles main website addresses differently.

File: src/pages/partials/general/_scripts.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/flickity@3.0.0/dist/flickity.pkgd.min.js"></script>
```
Comment (mahdiyazdani) [inline src/pages/partials/general/_scripts.html:2]: Please remove unused static resources.

File: src/pages/partials/general/_meta.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+<meta name="keywords" content="${props.keywords}" />
+<meta name="description" content="${props.description}" />
+<meta name="author" content="MyPreview" />
+<meta name="copyright" content="© 2015 - 2022 MyPreview. All Rights Reserved." />
+<meta property="og:locale" content="en_US">
+<meta property="og:type" content="website">
+<meta property="og:site_name" content="Plugin name">
+<meta property="og:title" content="${props.title}">
+<meta property="og:description" content="${props.description}">
+<meta property="og:url" content="https://mypreview.github.io/plugin-name/${props.slug}">
```
[the remark this answers is not in the document]
Comment (gooklani) [reply src/pages/partials/general/_meta.html:13]: The landing page slug will be set empty `""` but inner pages need to have a slug, for example, `docs`.

**Review by mahdiyazdani — APPROVED** [review]
Thank you for the new structure! Great work 🚀

**Pull request merged**